test/integration_test/common_test.go

```diff
@@ -816,21 +817,73 @@ func scheduleBidirectionalClusterPair(cpName, cpNamespace, projectMappings strin
 	// Create source --> destination and destination --> cluster pairs using storkctl
 	factory := storkctl.NewFactory()
 	cmd := storkctl.NewCommand(factory, os.Stdin, os.Stdout, os.Stderr)
-	cmd.SetArgs([]string{"create", "clusterpair", "-n", cpNamespace, cpName,
+	cmdArgs := []string{"create", "clusterpair", "-n", cpNamespace, cpName,
 		"--src-kube-file", srcKubeconfigPath,
-		"--src-ip", srcInfo[clusterIP],
-		"--src-token", srcInfo[tokenKey],
+		"--src-ep", fmt.Sprintf("%s:%s", srcInfo[clusterIP], srcInfo[clusterPort]),
```

Comment:
Even this is auto detected. Though we support it to override with user given values. lets use the default behavior and not pass "--src-ep" and "--dest-ip" with storkctl commanf.

Reply:
Fixed.